packages/shopware-6-client/src/services/checkoutService.ts

```diff
@@ -54,6 +54,16 @@ export async function handlePayment(
   if (!params?.orderId) {
     throw new Error("handlePayment method requires orderId");
   }
+  
+   /**
+   * save contextToken in sessionStorage when using Webkit
+   * https://github.com/vuestorefront/shopware-pwa/pull/1817
+   */
+  if(navigator.userAgent.indexOf('WebKit')){
```

Comment:
```suggestion
  if(navigator?.userAgent.includes('WebKit')){
```
just to be sure on server side, as API client can be used there as well